Library/Homebrew/cmd/doctor.rb

```diff
@@ -444,7 +444,7 @@ def check_access_usr_local
     The /usr/local directory is not writable.
     Even if this directory was writable when you installed Homebrew, other
     software may change permissions on this directory. Some versions of the
-    "InstantOn" component of Airfoil are known to do this.
+    "InstantOn" component of Airfoil or running Cocktail cleanup/optimizations are known to do this.
```

Comment:
:+1: but please wrap this at 80 characters so it displays nicely on a default terminal, thanks!